libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -316,9 +316,24 @@ void AP_InertialSensor_Backend::_notify_new_accel_raw_sample(uint8_t instance,
         _sem->give();
     }
 
+    if(instance==0)
+    {
+    	_fourier_analysis.accumulate_discrete(accel, dt, _imu._gyro[instance]);
+    }
+
     log_accel_raw(instance, sample_us, accel);
 }
 
+Vector2f AP_InertialSensor_Backend::get_pitch_yaw_FT(void)
+{
+	return _fourier_analysis.get_result();
+}
+
+void::AP_InertialSensor_Backend::synchronize_fourier_phase(float actual_heading)
```

Comment:
void:: ?

Reply:
yes, why not?

the synchronize_fourier_phase is updating the Fourier_Analysis objects with latest compass value. It does not return anything

Reply:
the problem is not void, it is :: after void